yarn.lock

```diff
@@ -1184,8 +1184,10 @@
     i18next-chained-backend "^0.1.2"
     i18next-xhr-backend "^1.5.1"
 
-"@jahia/search-ui-jahia-connector@../search-ui-jahia-connector":
+"@jahia/search-ui-jahia-connector@^2.0.0":
```

Comment:
Shouldn't it be 2.0.1?

Reply:
it should pick up the latest one automatically with "^" which will be 2.0.1 once it's available